splunk-otel-android/build.gradle

```diff
@@ -63,4 +63,72 @@ dependencies {
     testImplementation 'androidx.test:core:1.3.0'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
+}
+
+publishing {
+    repositories {
+        maven {
+            def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2"
+            def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshot"
+            url = project.hasProperty('release') ? releasesRepoUrl : snapshotsRepoUrl
```

Comment:
technically/pedantically iffy because `-Prelease=false` would trigger the wrong url here but the version would have `-SNAPSHOT` on it.  Hmmm...probably not an issue.

Reply:
I cleaned it up